RHEL/6/input/checks/avahi_check_ttl.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
+  <definition class="compliance" id="avahi_check_ttl" version="1">
+    <metadata>
+      <title>Check Avahi Responses' to TTL Field</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 6</platform>
+      </affected>
+      <description>Check that Avahi is ignoring packets
+      unless the TTL field is 255.
+      </description>
+      <reference source="galford" ref_id="20150316" ref_url="test_attestation" />
+    </metadata>
+    <criteria comment="Check Avahi TTL setting" operator="OR">
+      <extend_definition comment="avahi package is not installed" definition_ref="package_avahi_installed" negate="true" />
+      <criterion comment="Avahi TTL setting is enabled" test_ref="test_avahi_ttl_setting" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="Avahi TTL setting" id="test_avahi_ttl_setting" version="1">
+    <ind:object object_ref="object_avahi_ttl_setting" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object comment="Avahi TTL setting" id="object_avahi_ttl_setting" version="1">
+    <ind:filepath>/etc/avahi/avahi-daemon.conf</ind:filepath>
+    <ind:pattern operation="pattern match">^[\s]*check\-response\-ttl[\s]*=[\s]*yes$</ind:pattern>
```

Comment:
avahi-daemon doesn't seem to accept space characters between config key & config value. For example using \t or ' ' (space) leads to:
```
# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 6.6 (Santiago)
# rpm -q avahi
avahi-0.6.25-15.el6.x86_64
# cat -A /etc/avahi/avahi-daemon.conf | grep ttl
check-response-ttl =yes$
# service avahi-daemon restart
Shutting down Avahi daemon:                                [FAILED]
Starting Avahi daemon... Invalid configuration key "check-response-ttl " in group "server"

                                                           [FAILED]
```
Using '\t' ditto:
```
# cat -A /etc/avahi/avahi-daemon.conf | grep ttl
check-response-ttl^I=yes$
# service avahi-daemon restart
Shutting down Avahi daemon:                                [FAILED]
Starting Avahi daemon... Invalid configuration key "check-response-ttl	" in group  "server"                                                            [FAILED]
```
But leading '\t' (\t at start of the line is OK):
```
# cat -A /etc/avahi/avahi-daemon.conf | grep ttl
^Icheck-response-ttl=yes$
# service avahi-daemon restart
Shutting down Avahi daemon:                                [FAILED]
Starting Avahi daemon...                                   [  OK  ]
```
Leading space is OK too:
```
 cat -A /etc/avahi/avahi-daemon.conf | grep ttl
  check-response-ttl=yes$
# service avahi-daemon restart
Shutting down Avahi daemon:                                [  OK  ]
Starting Avahi daemon...                                   [  OK  ]
# service avahi-daemon status
avahi-daemon (pid  3322) is running...
```
=> Please drop the [\s]* expressions around the equal ('=') character (in this OVAL check & all the ones below too). Otherwise looks good to me.